base/ca/shared/profiles/ca/estServiceCert.cfg

```diff
@@ -6,7 +6,8 @@ desc=EST service certificate profile
 enable=true
 input.i1.class_id=certReqInputImpl
 input.i2.class_id=submitterInfoInputImpl
-input.list=i1,i2
+input.i3.class_id=clientAuthInfoInputImpl
+input.list=i1,i2,i3
```

Comment:
Since EST is officially supported in PKI 11.6, we probably should provide an upgrade script to apply these changes to existing instances.

Reply:
Not sure if we have to migrate. The initial goal is to make possible to define policies in the CA for the certificate but it should not be mandatory and current setup should work without changes.

If we decide to drop the current EST subsystem policies and configure only in the CA than we must implement a migration. Therefore, we can decide in a following PR.

Reply:
It is something to consider.  The original implementation did not allow for agent-approved enrollments.